wiki/Client/File_formats/Osu_(file_format)/en.md

```diff
@@ -4,346 +4,623 @@
-| `Mode` | Integer | Game mode (`0` = osu!, `1` = osu!taiko, `2` = osu!catch, `3` = osu!mania) | 0 |
-| `LetterboxInBreaks` | 0 or 1 | Whether or not breaks have a letterboxing effect | 0 |
-| `StoryFireInFront` | 0 or 1 | *Deprecated* | 1 |
-| `UseSkinSprites` | 0 or 1 | Whether or not the storyboard can use the user's skin images | 0 |
-| `AlwaysShowPlayfield` | 0 or 1 | *Deprecated* | 0 |
-| `OverlayPosition` | String | Draw order of hit circle overlays compared to hit numbers (`NoChange` = use skin setting, `Below` = draw overlays under numbers, `Above` = draw overlays on top of numbers) | NoChange |
+| `PreviewTime` | Integer | Time in milliseconds when the audio preview should start on the [song selection screen](/wiki/Client/Interface#song-select) | `-1` |
+| `Countdown` | Integer | Speed of the [countdown](/wiki/Beatmapping/Countdown) before the first hit object | `1` |
+| `SampleSet` | String | Sample set that will be used if timing points do not override it | `Normal` |
+| `StackLeniency` | Decimal | Multiplier for the threshold in time where hit objects placed close together [stack](/wiki/Mapping_techniques/Stack) (0–1) | `0.7` |
+| `Mode` | Integer | Game mode | `0` |
+| `LetterboxInBreaks` | 0 or 1 | Whether or not breaks have a letterboxing effect | `0` |
+| `StoryFireInFront` | 0 or 1 | *Deprecated* | `1` |
+| `UseSkinSprites` | 0 or 1 | Whether or not the storyboard can use the user's skin images | `0` |
+| `AlwaysShowPlayfield` | 0 or 1 | *Deprecated* | `0` |
+| `OverlayPosition` | String | Draw order of hit circle overlays compared to [hit numbers](/wiki/Skinning/Interface#countdown) | `NoChange` |
 | `SkinPreference` | String | Preferred skin to use during gameplay |  |
-| `EpilepsyWarning` | 0 or 1 | Whether or not a warning about flashing colours should be shown at the beginning of the map | 0 |
-| `CountdownOffset` | Integer | Time in beats that the countdown starts before the first hit object | 0 |
-| `SpecialStyle` | 0 or 1 | Whether or not the "N+1" style key layout is used for osu!mania | 0 |
-| `WidescreenStoryboard` | 0 or 1 | Whether or not the storyboard allows widescreen viewing | 0 |
-| `SamplesMatchPlaybackRate` | 0 or 1 | Whether or not sound samples will change rate when playing with speed-changing mods | 0 |
+| `EpilepsyWarning` | 0 or 1 | Whether or not a warning about flashing colours should be shown at the beginning of the map | `0` |
+| `CountdownOffset` | Integer | Speed of the countdown in [beats](/wiki/Beat) before the first hit object | `0` |
+| `SpecialStyle` | 0 or 1 | Whether or not the "N+1" style key layout is used for osu!mania | `0` |
+| `WidescreenStoryboard` | 0 or 1 | Whether or not the storyboard allows widescreen viewing | `0` |
+| `SamplesMatchPlaybackRate` | 0 or 1 | Whether or not sound samples will change rate when playing with speed-changing [mods](/wiki/Game_modifier) | `0` |
 
-## Editor
+### Notice
```

Comment:
would rather not have one heading like this for each section, because that would clutter the table of contents

with a lead paragraph before the tables and with the blockquotes removed, each section works quite well as its own heading

Reply:
Should i use "**" (bold) instood?

Reply:
no, the point isn't to replace the heading with bolding, it's to remove the heading